src/SSHDebugPS/Microsoft.SSHDebugPS.pkgdef

```diff
@@ -23,4 +23,17 @@
 "Assembly"="Microsoft.SSHDebugPS"
 "Class"="Microsoft.SSHDebugPS.Docker.DockerPortSupplier"
 "InprocServer32"="$WinDir$\SYSTEM32\MSCOREE.DLL"
-"CodeBase"="$PackageFolder$\Microsoft.SSHDebugPS.dll"
\ No newline at end of file
+"CodeBase"="$PackageFolder$\Microsoft.SSHDebugPS.dll"
+
+[$RootKey$\CLSID\{91BDF293-E6A0-49C4-B033-6F36CFC4FF98}]
+"Assembly"="Microsoft.SSHDebugPS"
+"Class"="Microsoft.SSHDebugPS.Docker.DockerPortPicker"
+"InprocServer32"="$WinDir$\SYSTEM32\MSCOREE.DLL"
+"CodeBase"="$PackageFolder$\Microsoft.SSHDebugPS.dll"
+
+[$RootKey$\RuntimeConfiguration\dependentAssembly\codeBase\{7E3052B2-FB42-4E38-B22C-1FD281BD4413}]
+"name"="Microsoft.SSHDebugPS"
+"publicKeyToken"=""
```

Comment:
Do we need this? It feels wrong to set this to empty string when in fact this dll has a public key token.

Reply:
@gregg-miskelly Since we have a different public key token for dev sign, do you know a way to do that? I'll test without that field set too.

---
In reply to: [287511218](https://github.com/microsoft/MIEngine/pull/868#discussion_r287511218) [](ancestors = 287511218)

Reply:
My hope was that we could just remove it. If not, I would probably suggest a comment.